src/plugins/intel_gpu/include/intel_gpu/plugin/plugin.hpp

```diff
@@ -58,6 +58,7 @@ class Plugin : public ov::IPlugin {
     std::shared_ptr<ov::ICompiledModel> import_model(std::istream& model,
                                                      const ov::SoPtr<ov::IRemoteContext>& context,
                                                      const ov::AnyMap& properties) const override;
+
```

Comment:
GPU plugin doesn't support import_model mmap, could we remove this unnecessary modification line?